Expose ps cluster (#117)
percona/percona-server-mysql-operator: source/expose.rst

Exposing cluster
================

The Operator provides entry points for accessing the database by client
applications in several scenarios. In either way the cluster is exposed with
regular Kubernetes `Service objects<https://kubernetes.io/docs/concepts/services-networking/service/>`_,
configured by the Operator.

This document describes the usage of :ref:`Custom Resource manifest options<operator.custom-resource-options>`
to expose the clusters deployed with the Operator. The expose options vary for

> different replication types: `Asyncronous<https://dev.mysql.com/doc/refman/8.0/en/replication.html>`_

Reviewer: ```suggestion
different replication types: `Asynchronous <https://dev.mysql.com/doc/refman/8.0/en/replication.html>`_
```

> and `Group Replicaiton <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.

Reviewer: ```suggestion
and `Group Replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.
```

> Asyncronous Replication

Reviewer: saw the same typo on multiple places

```suggestion
Asynchronous Replication
```

-----------------------

> With `Asyncronous or Semi-syncronous replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication-primary-secondary-replication.html>`_

Reviewer: ```suggestion
With `Asynchronous or Semi-synchronous replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication-primary-secondary-replication.html>`_
```

the cluster is exposed through a Kubernetes Service called
``<CLUSTER_NAME>-mysql-primary``: for example, ``cluster1-mysql-primary``.

.. image:: ./assets/images/exposure-async.svg
   :align: center

This Service is created by default and is always present. You can change the
type of the Service object by setting :ref:`mysql.primaryServiceType<mysql-primaryservicetype>`
variable in the Custom Resource.

> The following example exposes the Primary node of the asyncronous cluster with

Reviewer: ```suggestion
The following example exposes the Primary node of the asynchronous cluster with
```
